modules/web/src/app/core/services/wizard/wizard.ts

```diff
@@ -33,22 +33,15 @@ export class WizardService {
     constructor(private _parent: WizardService) {}
 
     handleProviderChange(provider: NodeProvider): void {
+      this._parent._steps = steps;
```

Comment:
We should call the setter method instead.
```suggestion
      this._parent.steps = steps;
```